types/async-rerun/tsconfig.json

```
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "esModuleInterop": true,
+        "noImplicitAny": true,
```

```suggestion
        "noImplicitAny": true,
```
please remove this option and make relevant changes
reference from README:
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

---

It's Done! Please review and merge it.
Thanks.